pkg/cmdlets/commit/commit.go

```diff
@@ -46,7 +46,11 @@ func PrintCommits(out output.Outputer, commits []*mono_models.Commit, orgs []gmo
 }
 
 func commitDataFromCommit(commit *mono_models.Commit, orgs []gmodel.Organization) (commitData, error) {
-	username, err := usernameForID(commit.Author, orgs)
+	if commit.Author == nil {
+		return commitData{}, locale.NewError("err_commit_no_author", "Commit has no author")
+	}
+
+	username, err := usernameForID(*commit.Author, orgs)
```

Comment:
We should just use an "anonymous" username rather than fail outright. We support anonymous commits, so this code shouldn't be failing if you're anonymous.